src/commands/buildImageInAzure/IBuildImageContext.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ContainerRegistryManagementClient, Run as AcrRun } from '@azure/arm-containerregistry';
+import * as vscode from 'vscode';
+import { IContainerAppContext } from '../createContainerApp/IContainerAppContext';
+import { IDeployFromRegistryContext } from '../deploy/deployFromRegistry/IDeployFromRegistryContext';
+import { Item } from "./DockerFileItemStep";
+
+export interface IBuildImageContext extends IContainerAppContext, IDeployFromRegistryContext {
```

Comment:
Some potential suggestions:

1) Move `buildImageInAzure/` to sit under `deploy/` (related to all the deploy image code)

2) Instead of extending `IContainerAppContext` and `IDeployFromRegistryContext`, extend from `IDeployBaseContext`

3) Rename `IBuildImageContext` to match the directory name.  So something like: `IBuildImageInAzureContext`